### PR TITLE
V3-1307 combine first and last name to full name

### PR DESCRIPTION
**Pull request by crawter**

### CODE REVIEW

File: satellite/console/consoleweb/consoleql/mutation.go [hunk 1]
```diff
@@ -283,12 +288,17 @@ func rootMutation(service *console.Service, mailService *mailservice.Service, ty
...
 					for _, user := range users {
+						userName := user.ShortName
+						if user.ShortName == "" {
+							userName = user.FullName
+						}
+
 						err = mailService.SendRendered(
 							p.Context,
-							[]post.Address{{Address: user.Email, Name: fmt.Sprintf("%s %s", user.FirstName, user.LastName)}},
+							[]post.Address{{Address: user.Email, Name: fmt.Sprintf("%s %s", user.FullName, user.ShortName)}},
```
Comment (rikysya) [inline satellite/console/consoleweb/consoleql/mutation.go]: What is the meaning of combining Fullname and Shortname?

Comment (jtolds) [reply satellite/console/consoleweb/consoleql/mutation.go]: not all cultures have the concept of first name and last name. the only general thing that works well across all cultures is full name and preferred short name

File: satellite/satellitedb/projectmembers.go [hunk 2]
```diff
@@ -48,8 +48,8 @@ func (pm *projectMembers) GetByProjectID(ctx context.Context, projectID uuid.UUI
 				INNER JOIN users u ON pm.member_id = u.id
 					WHERE pm.project_id = ?
 					AND ( u.email LIKE ? OR 
-						  u.first_name || u.last_name LIKE ? OR
-						  u.last_name || u.first_name LIKE ? )
+						  u.full_name LIKE ? OR
```
Comment (Barterio) [inline satellite/satellitedb/projectmembers.go]: Assume you have 
u.full_name = 'Bohdan Artemenko', 
search_query = 'artemenko'
Will this code return 'Bohdan Artemenko' user?

File: satellite/console/consoleweb/consoleql/mutation.go (hunk 1 above)
Comment (rikysya) [reply satellite/console/consoleweb/consoleql/mutation.go]: `Name: fmt.Sprintf("%s %s", user.FullName, user.ShortName)}}`
I mean for email recipient name it should be one of the above but not both combined

File: satellite/satellitedb/projectmembers.go (hunk 2 above)
Comment (crawter) [reply satellite/satellitedb/projectmembers.go]: @Barterio no, we should change this query, we will create a bug in our backlog

**Pull request merged**